src/components/runtime-common.js

```diff
@@ -98,7 +100,7 @@ export const AuthenticatedCookieSetter = () => {
 
 export const PeriodicCookieSetter = () => {
   const signal = Utils.useCancellation()
-  Utils.usePollingEffect(
+  getLocalPref(cookiesAcceptedKey) !== false && Utils.usePollingEffect(
```

Comment:
An effect shouldn't happen conditionally. Instead, conditionalize the rendering of this component where it's used.